### PR TITLE
Added docker image for rpi

### PR DESCRIPTION
**Pull request by danielo515**
- Added instructions for build rpi image
- Added npm scripts for building images

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,7 +73,9 @@
     "test:coverage": "nyc --reporter=html --reporter=text mocha -R spec ./test/functional ./test/unit",
     "test-travis": "eslint . && npm run test:coverage",
     "test-only": "mocha ./test/functional ./test/unit",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "build-docker": "docker build -t verdaccio .",
```
Comment (juanpicado) [inline package.json]: docker cli comes from external software. I'm right?

Comment (danielo515) [inline package.json]: Yes, it expects the docker to be installed on the machine. This is also required if you want to follow the current instructions.

Comment (juanpicado) [inline package.json]: Could you write down that into the README? Mostly for general understanding.

Comment (danielo515) [inline package.json]: Readme updated.

**Comment by juanpicado** [comment]
Thanks @danielo515 👍

**Comment by DanielRamosAcosta** [comment]
Where is uploaded the latest rpi image? I can't find it at docker hub... 😢

**Comment by juanpicado** [comment]
@DanielRamosAcosta Actually we are not shipping that image yet because until today nobody asked for it :) But, now you are asking, I have no clue since we have already one Dockerfile. Any idea?

**Pull request merged**